Note behaviour of negative commit timeout in 6.x (#42834)
elastic/elasticsearch: docs/reference/modules/discovery/zen.asciidoc

other nodes in the cluster. Each node receives the publish message, acknowledges
it, but does *not* yet apply it. If the master does not receive acknowledgement
from at least `discovery.zen.minimum_master_nodes` nodes within a certain time
(controlled by the `discovery.zen.commit_timeout` setting which defaults to 30

> seconds and negative values are treated as 0 second) the cluster state change is

Reviewer: ```suggestion
seconds, with negative values are treated as 0 seconds) the cluster state change is
```
